service/src/main/java/bio/terra/catalog/controller/DatasetApiController.java

```diff
@@ -20,22 +21,22 @@
 
 @Controller
 public class DatasetApiController implements DatasetsApi {
-  private final HttpServletRequest request;
+  private final HttpServletRequest servletRequest;
```

Comment:
This is an unrelated change to fix a warning in IntelliJ, where a method parameter with the same name (`request`) shadows the field.